modules/microshift-adding-containers-to-blueprint.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+= Adding the {product-title} container images
+You can embed {product-title}'s container images into the {op-system-ostree} images, so that they are immediately available to the CRI-O container engine after booting, without having to pull the images over the network from a container registry. In Image Builder, a container image is embedded by adding a reference to it to the Image Builder blueprint.
+
+.Example adding a container image to a blueprint:
+[source,terminal]
+----
+[[containers]]
+source = "<image_pullspec_with_tag_or_digest>"
+----
+
+To embed {product-title}'s container images, you need to know the exact list of container image references used by the {product-title} version you are embedding. You can get this list from the `microshift-release-info` RPM package matching the version of the `microshift` RPM in your blueprint using the following procedure.
+
+You need to have the exact list of container image references used by the {product-title} version to embed {product-title} 's container images. Use the `microshift-release-info` RPM package matching the version of the `microshift` RPM in your blueprint.
```

Comment:
This paragraph kind of repeats and rephrases the previous one. These two should be merged.